scripts/repo_health_status_report/output_health_report.py

```diff
@@ -23,11 +23,10 @@
 from ci_tools.parsing import ParsedSetup
 from ci_tools.environment_exclusions import (
     is_check_enabled,
-    IGNORE_FILTER,
     IGNORE_PACKAGES,
-    FILTER_EXCLUSIONS,
 )
-
+IGNORE_FILTER = ["nspkg", "mgmt", "cognitiveservices"]
```

Comment:
Should we -- similar to with `IGNORE_PACKAGES` -- append to the list from `environment_exclusions` instead of redefining it here, so we automatically get any new additions there?

Reply:
Good idea!